examples/bert/README.md

```diff
@@ -12,6 +12,30 @@ Go to [How to run](#how-to-run)
 
 
 ## Optimization Workflows
+### BERT Quantization QDQ
+This workflow quantizes the model. It performs the pipeline:
+- *HF Model-> ONNX Model ->Quantized Onnx Model*
+
+Config file: [Intel/bert-base-uncased](bert_ptq_qdq.json)
+
+#### Accuracy / latency

```

Comment:
From last pr i forgot to add throughput here. Can you update this to `Accuracy / latency / Throughput`?